daprdocs/content/en/developing-applications/building-blocks/state-management/query-state-store/query-redis-store.md

```diff
@@ -49,18 +59,21 @@ To get the state version/ETag, use the command:
 HGET myapp||balance version
 ```
 
-## 4. Read actor state
+## Read actor state
 
 To get all the state keys associated with an actor with the instance ID "leroy" of actor type "cat" belonging to the application with ID "mypets", use the command:
 
 ```bash
 KEYS mypets||cat||leroy*
 ```
 
-And to get a specific actor state such as "food", use the command:
+To get a specific actor state such as "food", use the command:
 
 ```bash
 HGET mypets||cat||leroy||food value
 ```
 
-> **WARNING:** You should not manually update or delete states in the store. All writes and delete operations should be done via the Dapr runtime.
+{{% alert title="Warning" color="warning" %}}
+You should not manually update or delete states in the store. All writes and delete operations should be done via the Dapr runtime.
```

Comment:
```suggestion
You should not manually update or delete states in the store. All writes and delete operations should be done via the Dapr runtime. The one exception to this, is that it is often required to delete all the actor records in a state store, _once you know that these are no longer in use_, to prevent a build up of unused actor instance that may never be loaded again.
```